src/Keboola/StorageApi/Client.php

```diff
@@ -2431,7 +2431,7 @@ public function apiDelete($url)
     /**
      * @return mixed|string
      */
-    public function apiDeleteParamsJson(string $url, array $data = [])
+    public function apiDeleteJson(string $url, array $data = [])
```

Comment:
Tohle je docela BC break, pokud to někdo použil - je to public metoda

Reply:
jo, jasne 🤦 vratim zpatky

Reply:
Můžeš to udělat tak, že v metodě apiDeleteParamsJson uděláš jen proxy na tu novou metodu a tu starou dáš jako `deprecated, will be removed 2023`.